quipucords/api/source/view.py

```diff
@@ -116,6 +117,26 @@ def create(self, request, *args, **kwargs):
 
         # Create expanded host cred JSON
         expand_credential(json_source)
+
+        # check to see if a connection scan was requested
+        # through query parameter
+        scan = request.query_params.get('scan', False)
```

Comment:
what happens if someone provides `?scan=ImNotABoolean`?  I feel like you could add a check for is_boolean, similar to: https://github.com/quipucords/quipucords/blob/master/quipucords/api/common/util.py#L21

We should throw a Validation error if the value isn't True or False